sdks/go/pkg/beam/core/runtime/xlangx/expansionx/process_test.go

```diff
@@ -22,15 +22,28 @@ import (
 	"testing"
 )
 
+func TestFindOpenPort(t *testing.T) {
+	port, err := findOpenPort()
+	if err != nil {
+		t.Fatalf("failed to find open port, got %v", port)
```

Comment:
typo: port is being printed instead of err.

Reply:
Good catch!